charts/authelia/templates/deployment.yaml

```diff
@@ -45,13 +45,13 @@ spec:
       labels: {{ include "authelia.labels" (merge (dict "Labels" .Values.pod.labels) .) | nindent 8 }}
       annotations:
         {{- if (include "authelia.generate.configMap" .) }}
-        authelia.com/checksum/config: {{ include (print $.Template.BasePath "/configMap.yaml") . | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/configMap.yaml") . | sha256sum }}
         {{- end }}
         {{- if (include "authelia.generate.acl.secret" .) }}
-        authelia.com/checksum/acl: {{ include (print $.Template.BasePath "/secretACL.yaml") . | sha256sum }}
+        checksum/acl: {{ include (print $.Template.BasePath "/secretACL.yaml") . | sha256sum }}
```

Comment:
```suggestion
        authelia.com/acl-checksum: {{ include (print $.Template.BasePath "/secretACL.yaml") . | sha256sum }}
```

Reply:
Yep, these work too - its what i tried first before looking at what the master branch had.